plugins/modules/ansible_galaxy_install.py

```diff
@@ -239,12 +249,18 @@ def process(rc, out, err):
             version = tuple(int(x) for x in version.split('.')[:3])
             return version
 
-        with self.runner("version", check_rc=True, output_process=process) as ctx:
-            return ctx.run(version=True)
+        try:
+            runner = self._make_runner("C.UTF-8")
+            with runner("version", check_rc=False, output_process=process) as ctx:
+                return runner, ctx.run(version=True)
+        except UnsupportedLocale as e:
+            runner = self._make_runner("en_US.UTF-8")
+            with runner("version", check_rc=True, output_process=process) as ctx:
+                return runner, ctx.run(version=True)
```

Comment:
What will happen when the `UnsupportedLocale` exception is raised here another time? Will the module just crash, or how will it react?

Reply:
Being a ModuleHelper, any exception will be captured and `module.fail_json()` accordingly.